### DISCUSSION TITLE
List start Prop

### DISCUSSION THREAD

**Discussion by elibroftw** [mantinedev/mantine#3477] [Q&A] [answered]
Allow Lists to start ordering at a number other than 1.

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_ol_start

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/eloquent-haze-l7clcy?file=/src/App.tsx

**Reply by elibroftw** [reply]
Thanks
